pykickstart/options.py

```diff
@@ -74,6 +74,16 @@ def mountpoint(value):
 
     return value
 
+def uid_gid(value):
+    try:
+        number = int(value)
+        if number > 2**32-1 or number < 1:
+            raise ArgumentTypeError(_("UID or GID value out of permitted range: %r") % value)
+        return number
+    except (ValueError, TypeError):
```

Comment:
Aside from the larger implications of this, I am interested in this bit - Do I need to allow None here?

Reply:
I think that `uid_gid` shoudn't be called when `--gid` (or `--uid`) is not specified in the kickstart file. You can check the documentation of the argument parser.